.gitignore

```diff
@@ -11,5 +11,8 @@
 .egdist
 *.sublime-*
 
-#build directory
+# Ignore all files in the build directory
+build/*
```

Comment:
seems repetative

Reply:
This probably happened when fixing merge conflicts. I've fixed it now.